content/reference/performance/availability.md

```diff
@@ -2,16 +2,86 @@
 title: Availability
 ---
 
-Dolt's availability story is similar to a single Postgres or MySQL instance, and read-only replicas are on our roadmap. Dolt's multi-instance high availability provides weaker consistency in exchange for total process isolation.
+Overview:
 
-### Single Instance
+- [Backups](#backups)
+- [Read Replication](#read-replication)
+- [Failover](#failover)
+- [Multi-Master](#multi-master)
 
-Dolt's single instance availability model is similar to a single instance of MySQL or Postgres. We are constrained by the performance of our query engine. We do not currently support read-only replicas, but that feature is on our roadmap.
+In the context of an OLTP database, availability is the ability
+to tolerate and recover from server failures. Durability, or storing backups of
+data, is a basic prerequisite of availability. The more useful version
+of availability involves servers communicating changes with one another,
+load balancing read requests, and maintaining a responsive application
+when servers fail.
 
-### Multi Instance
+## Backups
 
-Dolt multi-instance is completely decentralized, and new application instances each have their own server instance. New application instances obtain a server by cloning a remote and launching a Dolt SQL Server. Dolt's `pull` and `push` semantics provide a way to periodically reconcile changes. The architecture looks something like this:
+Storing snapshots of your database is technically a durability property
+(the "D" in ACID). But not losing data is required for database
+availablity.
 
-![Multi-instance Dolt](../../.gitbook/assets/dolt-high-availability-multi-instance.png)
+Dolt has remotes, which backs up some data. Dolt backup snapshot the
```

Comment:
shapshots

Reply:
What about this?

Dolt has two options for backup: remotes and backups. 

Using remotes for backups should be suitable for some use cases. Pushing to a remote creates an off server copy of the branch being pushed. Frequent pushing to a remote can serve as a reasonable backup.

Dolt also has backups, served by the `dolt backup` command. These backups look more like traditional database backups. The entire state of the database, including uncommitted changes, are copied to another location.